Profiles yml precedence core fusion (#8443)
fishtown-analytics/docs.getdbt.com: website/snippets/_connection-profiles.md

## Advanced: Customizing a profile directory

<Tabs>

> <TabItem value="core" label="dbt Core">

Reviewer: Non-blocker, the tab order is inconsistent between the Connection profiles page and the `profiles.yml` page. The Connection profiles page shows dbt Core first and dbt Fusion second, while the `profiles.yml` page shows dbt Fusion first and dbt Core second. I suggest making the order consistent across both pages (maybe dbt Fusion first?)

Author: good call!

dbt Core determines the parent directory for `profiles.yml` using the following precedence:

1. `--profiles-dir` option
2. `DBT_PROFILES_DIR` environment variable
3. current working directory
4. `~/.dbt/` directory

</TabItem>
<TabItem value="fusion" label="dbt Fusion">

dbt Fusion determines the parent directory for `profiles.yml` using the following precedence:

1. `--profiles-dir` option

> 2. current working directory

Author: edit

3. `~/.dbt/` directory

> Note that <Constant name="fusion"/> doesn't currently support the `DBT_PROFILES_DIR` environment variable or setting the `profiles.yml` at the current working directory. 

Reviewer: ```suggestion
Note that <Constant name="fusion"/> doesn't currently support the `DBT_PROFILES_DIR` environment variable or setting the `profiles.yml` in the current working directory. 
```

If using this method, the `--profiles-dir` option needs to be provided every time you run a dbt command.

### 2. Use the `DBT_PROFILES_DIR` environment variable to change the default location (dbt Core only)

> Specifying this environment variable overrides the directory that dbt Core looks for your `profiles.yml` file in. You can specify this by running:

Reviewer: ```suggestion
Setting this environment variable tells dbt Core to look for your `profiles.yml` file in the specified directory instead of the default location. You can specify this by running:
```

> Setting this environment variable tells dbt Core to look for your `profiles.yml` file in the specified directory instead of the default location. You can specify this by running:

Reviewer: ```suggestion
Setting this environment variable tells <Constant name="core" /> to look for your `profiles.yml` file in the specified directory instead of the default location. You can specify this by running:
```

```
$ export DBT_PROFILES_DIR=path/to/directory
```

> **Note:** This environment variable is not supported in dbt Fusion.

Author: ```suggestion
Note: This environment variable isn't supported in <Constant name="fusion"/>.
```
